tests/wrapper/test_simulation_objects.py

```diff
@@ -37,7 +38,7 @@ def test_update_state(self, traffic_update):
     def test_accessible_edges(self):
         signal = Signal("fancy-signal")
         edges = [Edge("a"), Edge("b"), Edge("a-re"), Edge("b-re")]
-        signal._edges = edges
+        signal._edges = edges  # pylint: disable=protected-access
```

Comment:
Maybe we should disable protected-access completely for the tests?

Reply:
We could discuss this tomorrow in the daily